shesha-core/src/Shesha.Framework/Otp/IOtpManager.cs

```diff
@@ -9,17 +9,17 @@ public interface IOtpManager
         /// <summary>
         /// Send one-time-pin
         /// </summary>
-        Task<ISendPinResponse> SendPinAsync(ISendPinInput input);
+        Task<ISendPinResponse> SendPinAsync(SendPinInput input);

```

Comment:
Hi @OmolemoBlessingLethuloe. What was a reason of removal of interface usage here? If it's not required for your task please revert the change, I know one of devs are busy with enhancements of the OTP functionality and this change may complicate merge. Initially interfaces used here to make services more generic